ruby-challenges.rb

```diff
@@ -33,3 +60,13 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+ def pedal_faster
+    @current_speed = 10
+ end
+
+ def brake
+    @
+
+
+    # ----------------------------------- Extremely confusing, I dont understand how to get that expected output in the examples. Where/how would I incorporate the 'my_bike_.pedal' with the given information and how would I be able to get the different values for each expected output if I've already done one, how would I not interfare with the one I made? Before that when I ran the first part of my code I got this error: 'syntax error, unexpected ';', expecting '='' where would I put these syntax? I tried beside the parameter with both equal and semi colon and still got the same error
```

Comment:
Sorry this was confusing.  I encourage you to review the Ruby Classes, Objects, Inheritance section: https://github.com/learn-academy-2022-delta/Syllabus/blob/main/ruby/classes-objects.md  for a better understanding.   Let me know if you would like help reworking this problem.